MiniApp/Classes/APIRequest.swift

```diff
@@ -0,0 +1,2 @@
+internal class APIRequest {
```

Comment:
What is this class for? A base request that will be used when fetching the listing, manifest, etc?